config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -0,0 +1,38 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-azure:
+  - name: pull-cluster-api-provider-azure-build
+    always_run: true
```

Comment:
one last suggestion -- it might be worth making these `optional: true` or `skip_report: true` (prefer the first option imho) in order to not have them block PRs when first introduced, so if anything is unexpectedly broken this time y'all can keep working over there until these are happy.

when they look happy, file a follow-up to drop that option and make them blocking.

if you're confident, feel free to keep them this way and go for it! :-)

Reply:
`optional: true` would tell tide that it should not consider this status blocking
`skip_report: true` also implies this by way of not posting any status to github, which is quieter but also less visible

Reply:
Sounds great as the project is going through a lot of transitions now.